### PR TITLE
feat(docs): add how-to guide on etl pipelines using duckdb, python and task runners

### PR DESCRIPTION
**Pull request by shrutimantri**
Updates #1534

### CODE REVIEW

File: content/docs/15.how-to-guides/etl-pipelines.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
+title: ETL Pipelines in Kestra
+icon: /docs/icons/tutorial.svg
+stage: Intermediate
+topics:
+  - Integrations
+---
+
+Build ETL pipelines in Kestra.
```
Comment (wrussell1999) [inline content/docs/15.how-to-guides/etl-pipelines.md:9]:
```suggestion
Build ETL pipelines in Kestra using DuckDB, Python and Task Runners.
```

File: content/docs/15.how-to-guides/etl-pipelines.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    secretKeyId: "{{ secret('AWS_SECRET_KEY_ID') }}"
+    region: "eu-central-1"
+    from: "{{ outputs.get_orders_per_product_csv.uri }}"
+    bucket: "my_bucket"
+    key: "orders/orders_per_product" 
+```
+
+Similar Query tasks can be perfomed on different databases like Snowflake, Postgres, etc.
+
+## Using Python
```
Comment (wrussell1999) [inline content/docs/15.how-to-guides/etl-pipelines.md:106]: Can you add a link to how to get AWS keys? https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/

Comment (wrussell1999) [reply content/docs/15.how-to-guides/etl-pipelines.md:106]: maybe at the top to cover it for the whole article

File: content/docs/15.how-to-guides/etl-pipelines.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    secretKeyId: "{{ secret('AWS_SECRET_KEY_ID') }}"
+    region: "eu-central-1"
+    from: "{{ outputs.python_etl.outputFiles('orders_per_product.csv') }}"
+    bucket: "my_bucket"
+    key: "orders/orders_per_product"
+```
+
+## Using Batch Task Runners
+
+When the python scripts get more compute-intesive or memory-intensive, it is advised to run them on remote batch compute resources using Batch Task Runners.
```
Comment (wrussell1999) [inline content/docs/15.how-to-guides/etl-pipelines.md:171]: It's worth noting that the Docker task runner is remote. They still run locally, just in a container. You should change this example to use AWS Batch Runners or equivalent

**Review by wrussell1999 — CHANGES_REQUESTED** [review]

File: content/docs/15.how-to-guides/etl-pipelines.md (hunk 2 above)
Comment (shrutimantri) [reply content/docs/15.how-to-guides/etl-pipelines.md:106]: This is pretty basic, and every AWS user should be aware about it. Do you think we should add this link? Also, this is not even an AWS specific guide.

Comment (wrussell1999) [reply content/docs/15.how-to-guides/etl-pipelines.md:106]: I still think it's worth doing. It's quite an easy thing to clarify. Feel free to replace with a docs link.

Comment (shrutimantri) [reply content/docs/15.how-to-guides/etl-pipelines.md:106]: ok.. sure.. let me add a note at the top.

**Review by wrussell1999 — APPROVED** [review]

**Pull request merged**